### PR TITLE
Add get_network_interface method and default

### PR DESCRIPTION
**Pull request by jimboca**
Let me know what you guys think.  Criticism welcome!

### CODE REVIEW

**Comment by jimboca** [comment]
Can you guys review this?  I got annoyed putting the code in multiple nodeservers so added it here.

File: polyinterface/polyinterface.py [hunk 1]
```diff
@@ -201,6 +202,11 @@ def __init__(self, envVar=None):
         Interface.__exists = True
         self.custom_params_docs_file_sent = False
         self.custom_params_pending_docs = ''
+        try:
+            self.network_interface = self.get_network_interface()
+            LOGGER.info('Connect: Network Interface: {}'.format(self.network_interface))
+        except:
+            LOGGER.error('Failed to determin Network Interface', exc_info=True)
```
Comment (exking) [inline polyinterface/polyinterface.py]: typo (determine)

File: polyinterface/polyinterface.py [hunk 2]
```diff
@@ -559,6 +565,23 @@ def save_typed_params(self, data):
         message = { 'typedparams': data }
         self.send(message)
 
+    """
+        Returns the network interface which contains addr, broadcase, and netmask elements
```
Comment (exking) [inline polyinterface/polyinterface.py]: broadcase = broadcast

**Review by exking — APPROVED** [review]
I'm good with it, just remember you can't use that method on cloud instances

**Comment by jimboca** [comment]
Thanks @exking Typo's fixed.  Also, cause I was curious, I changed the travis test to cat the log and it actually does seem to work in the cloud, but probably not needed.  Although if it does work then WirelessTags NS could be run in the PGC?
https://travis-ci.org/UniversalDevicesInc/polyglot-v2-python-interface/builds/609066674

**Comment by jimboca** [comment]
Should I merge, or wait for @Einstein42 ?

**Pull request merged**